src/pages/Careers/Careers.tsx

```diff
@@ -36,7 +42,7 @@ function Careers() {
                   <Card.Title>E-MAIL</Card.Title>
                   <Card.Text>
                     <a href="mailto:carrers@xtremecars.com" className="formatLink">
```

Comment:
Correct the casing for formatLink

Reply:
it should be format-link, and check if the link effect to the css are getting applied as well

Reply:
change it to format-link here as well 😄

Reply:
Okay @pashedin